h/views/admin/groups.py

```diff
@@ -71,10 +71,16 @@ def post(self):
         def on_success(appstruct):
             """Create a group on successful validation of POSTed form data."""
 
-            organization = self.organizations[appstruct["organization"]]
+            organization = self.organizations.get(appstruct["organization"])
+
             # We know this user exists because it is checked during schema validation
             creator_userid = self.user_svc.fetch(
-                appstruct["creator"], organization.authority
+                appstruct["creator"],
+                (
+                    organization.authority
+                    if organization
+                    else self.request.default_authority
```

Comment:
If the user selected `-- None --` for the organization then `organization` will be `None`. In that case create a group belonging to the default authority.